src/discordcr/voice.cr

```diff
@@ -9,9 +9,12 @@ module Discord
   class VoiceClient
     UDP_PROTOCOL = "udp"
 
-    # The mode that tells Discord we want to send encrypted audio
+    # DEPRECATED: Discord now supports multiple encryption modes
     ENCRYPTED_MODE = "xsalsa20_poly1305"
```

Comment:
Why not remove this?

Reply:
Was leaving it in because I wasn't sure if this was considered to be part of the public API or if it needed to be phased out over a release cycle, etc.

I can remove it if it's considered to be for internal use only, which I would imagine it is.

Reply:
You can remove it.